__tests__/e2e/common-libs-e2e.fixed-versions.spec.ts

```diff
@@ -0,0 +1,33 @@
+import tempy from 'tempy';
+import { bundle } from '../../src';
+import fs from 'fs';
+
+describe('Common Libs - Fixed versions', () => {
```

Comment:
What `Fixed` means? 🤔 I would clarify what it means or remove it if that makes sense.

Reply:
fixed as in static, hard coded in appose to latest,
when you use "1.2.3" in package json it is a fixed version.